### ISSUE TITLE
Error when partfuncfmt not defined in Configuration File

### ISSUE THREAD

**Issue by erwanp** [radis/radis#78]
Error spotted by @stephanedelperugia

## Context

~/.radis
```
[DB_ISSUE_78]
info = ... 
path =
        PATH/TO/cdsd_hitemp_07
format = cdsd-hitemp
levelsfmt = radis
```
> Note that parfuncfmt is not declared

Minimal Code
```
s = calc_spectrum(2000, 2300,         # cm-1
                  molecule='CO2',
                  isotope='1,2,3',
                  pressure=1.01325,   # bar
                  Tvib=1700,           # K
                  Trot=1700,           # K
                  databank='DB_ISSUE_78',
                  )
```

## Expected Behavior

`parfuncfmt` is not declared --> we could expect that it either:
- fails immediatly
- or use the default version

## Current Behavior

Crashes later with:

```
  File "d:\github\radis\radis\lbl\base.py", line 2765, in get_parsum
    return self.get_partition_function_interpolator(molecule, iso, state)

  File "d:\github\radis\radis\lbl\loader.py", line 2408, in get_partition_function_interpolator
    assert isinstance(parsum, RovibParFuncTabulator)

AssertionError
```

## Possible Solution

- If ``None``, use default value (i.e., calculate with `hapi`) 
- or crash immediatly after loading the ~/.radis file.

**Comment by erwanp** [comment]
Easy fix for this one and #80 if anyone wants to submit a PR!

**Comment by gagan-aryan** [comment]
Hey @erwanp. I am unable to replicate the errors in the description above. These are the messages I am getting. 
![Screenshot from 2021-02-17 11-36-36](https://user-images.githubusercontent.com/62807661/108163732-7bc9b280-7115-11eb-8f69-8c9f8c4b5c8d.png)
Could you please help me around with this?

**Comment by erwanp** [comment]
Hello @gagan-aryan , thanks for taking this topic !

From the error it seems that the database file you're using is probably in the wrong range, therefore you have no spectral lines and the code stops before reaching the error of #78 . 

To make it easier, let's use one of the features introduced in 0.9.28 and directly download the CO database from HITEMP (same error message can be reproduced, and it's a lighter database than CO2) : 

1. First update your repo (`git pull`). There were some troubles with CO non-eq calculations. I just fixed them in #203 so you can focus on this. 
2. Then run this code : 
```python
from radis import calc_spectrum
calc_spectrum(2000, 2300,         # cm-1
                  molecule='CO',
                  isotope='1',
                  pressure=1.01325,   # bar
                  Tvib=1700,           # K
                  Trot=1700,           # K
                  databank='hitemp',
                  ).plot()
```
This *should work*, and calculate CO. It should also have register a "HITEMP-CO" database in your ~/.radis config file , similar to : 
```
[HITEMP-CO]
info = HITEMP CO lines (2.2-22148.3 cm-1) with TIPS-2017 (through HAPI) for partition functions and RADIS spectroscopic constants for rovibrational energies (nonequilibrium)
path = C:\Users\erwan\.radisdb\CO-05_HITEMP2019.h5
format = hitemp-radisdb
parfuncfmt = hapi
levelsfmt = radis
download_url = https://hitran.org/hitemp/data/bzip2format/05_HITEMP2019.par.bz2
download_date = 07 Feb 2021
wavenumber_min = 2.248764
wavenumber_max = 22148.273542
```

3. Now copy this and register a new database, with missing line `parfuncfmt`. Ex : 
```
[HITEMP-CO-#78]
info = HITEMP CO lines (2.2-22148.3 cm-1) with TIPS-2017 (through HAPI) for partition functions and RADIS spectroscopic constants for rovibrational energies (nonequilibrium)
path = C:\Users\erwan\.radisdb\CO-05_HITEMP2019.h5
format = hitemp-radisdb
levelsfmt = radis
download_url = https://hitran.org/hitemp/data/bzip2format/05_HITEMP2019.par.bz2
download_date = 07 Feb 2021
wavenumber_min = 2.248764
wavenumber_max = 22148.273542
```

4. Now re-run the code from 1. fetching the new database : 
```python
from radis import calc_spectrum
calc_spectrum(2000, 2300,         # cm-1
                  molecule='CO',
                  isotope='1',
                  pressure=1.01325,   # bar
                  Tvib=1700,           # K
                  Trot=1700,           # K
                  databank='HITEMP-CO-#78',
                  ).plot()
```
This *should fail* with the error you're expecting. 

From this you can move on and simply declare a better error message, or choose (and print) a default backup option !
